bin/node/runtime/src/lib.rs

```diff
@@ -940,6 +940,9 @@ impl pallet_mmr::Config for Runtime {
 parameter_types! {
 	pub const AssetDepositBase: Balance = 100 * DOLLARS;
 	pub const AssetDepositPerZombie: Balance = 1 * DOLLARS;
+	pub const AllowFreezing: bool = true;
```

Comment:
what's the point of adding these? is there some good reason by a chain might want to limit the functionality over all assets it hosts?

Reply:
This was requested in eariler reviews afaik. At first all this functionality was supported by we couldn't come to consensus.